README.md

```diff
@@ -7,6 +7,17 @@ providing an easy way to create import / export dataflow.
 
 [![Coverage Status](https://coveralls.io/repos/github/code-rhapsodie/dataflow-bundle/badge.svg)](https://coveralls.io/github/code-rhapsodie/dataflow-bundle)
 
+Dataflow uses a linear generic workflow in three parts (one reader, any number of steps, one or more writers).
```

Comment:
Dataflow uses a linear generic workflow in three parts:
- a reader
-  any number of steps
- one or more writers).